x-pack/plugins/alerting/server/rules_client/methods/get_alert_summary.ts

```diff
@@ -56,6 +56,8 @@ export async function getAlertSummary(
           start: parsedDateStart.toISOString(),
           sort: [{ sort_field: '@timestamp', sort_order: 'desc' }],
           end: dateNow.toISOString(),
+          // filter out execute-action event logs
+          filter: 'NOT event.action: execute-action && event.provider: alerting',
```

Comment:
I think this might have to be `AND` not `&&`

Reply:
🤦‍♀️ whoops https://github.com/elastic/kibana/pull/150088/commits/301d1e1121563d0f02bcf5b43af98d9fb0976e0a